.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
```

Comment:
Given the fact that the CI passes and the CI logs look as expected, I'm assuming this workflow file is correct.
On a preliminary look, I don't see anything wrong.